EN.yaml

```
@@ -670,26 +670,26 @@ SPECIALIST_SHOUTMESSAGE_POINTS_ADDED: You added {0} SP points!
 SPECIALIST_SHOUTMESSAGE_POINTS_RESET: SP points reset!
 SPECIALIST_SHOUTMESSAGE_RESURRECTED: Your Specialist Card soul has been resurrected!
 SPECIALIST_SHOUTMESSAGE_USE_NEEDED: You need to be transformed into a Specialist to do this!
-SUM_MESSAGE_FAILED: Summing failed.
-SUM_MESSAGE_SUCCESS: Summing succeeded!
+SUM_MESSAGE_FAILED: Combination failed.
+SUM_MESSAGE_SUCCESS: Combination successfull!
 TIMESPACE_CHATMESSAGE_EASY_MODE: The time-space is in easy mode - the duration has been extended and you will not receive selected rewards.
 TIMESPACE_CHATMESSAGE_GATE_LOCKED: The gate is locked!
 TIMESPACE_CHATMESSAGE_LOW_LEVEL: My level is too low to join this time-space.
 TIMESPACE_CHATMESSAGE_NOT_FINISHED_TASKS: Not all time-space missions have been completed.
-TIMESPACE_CHATMESSAGE_ONLY_TEAM_LEADER_START: Only the party leader can start the TimeSpace
+TIMESPACE_CHATMESSAGE_ONLY_TEAM_LEADER_START: Only the party leader can start the TimeSpace.

```

Should be "Time-Space", not TimeSpace.
You could also change every "time-space", "Time-space", "TimeSpace" "timespace" etc. to "Time-Space" for consistency.

---

Its most frequent use is time-space so
```suggestion
TIMESPACE_CHATMESSAGE_ONLY_TEAM_LEADER_START: Only the party leader can start the time-space.
```

---

But the name is supposed to be Time-Space, not time-space. It's "Time-Space" almost everywhere in client texts, it's only been wrongly written here in server files.